doc/Analytics.xml

```diff
@@ -749,14 +724,21 @@
     </tt:Object>
   </tt:Frame>
 ]]></programlisting>
+        </section>
+        <section>
+          <title>Object Identifier</title>
+          <para>Objects are enumerated via their integer ObjectId. ObjectIds are implicitly created
+          by an analytics module with the first appearance of the object. A unique identifier can be
+          assigned to objects via the UUID attribute for linking object information from multiple
+          analytics module streams.</para>
```

Comment:
I'm worried it is a bit too specific and device manufacturers might not see the interest in implementing it.
I would add "for example to link object information...". The uniqueness is also very useful for consumers of the stream